client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -310,6 +313,7 @@ class ActivityLogItem extends Component {
 	renderSummary() {
 		const {
 			allowRestore,
```

Comment:
Perhaps we no longer need `allowRestore`, and instead overload the `disableRestore` prop for cases where rewind is not available?